styles/components/flexbox-hero.scss

```diff
@@ -0,0 +1,7 @@
+.flexbox-hero {
+  display: flex;
+  justify-content: center;
+  height: 40vh;
+  padding-top: 5rem;
```

Comment:
@rtablada i tried to add negative margins and calc > 100% and couldn't get the image to expand to the edges.